app/Jobs/Emails/PresentationSubmissions/SelectionProcess/PresentationSpeakerSelectionProcessAcceptedOnlyEmail.php

```diff
@@ -35,23 +38,30 @@ protected function getEmailEventSlug(): string
      * @param Summit $summit
      * @param SummitRegistrationPromoCode $promo_code
      * @param PresentationSpeaker $speaker
-     * @param string $speaker_role
+     * @param string $confirmation_token
      */
     public function __construct
     (
         Summit $summit,
         SummitRegistrationPromoCode $promo_code,
         PresentationSpeaker $speaker,
-        string $speaker_role,
         string $confirmation_token
     )
     {
         parent::__construct($summit, $speaker, $promo_code);
 
         $summit = $promo_code->getSummit();
```

Comment:
this line is not needed summit is already a param

Reply:
done